### ISSUE TITLE
Work around Firefox problems related to clearing date fields

### ISSUE THREAD

**Issue by craigfowler** [csf-dev/CSF.Screenplay.Selenium#2]
On the latest Firefox, it is now impossible to clear the value from a date field. This is shown by [a failing build] with the appropriate information:

[a failing build]: https://travis-ci.org/csf-dev/CSF.Screenplay/jobs/328755580

This has been addressed in the short term by ignoring the test in Firefox (in another ticket).

## Work to be done
A long term fix must be created:
* Investigate to determine the range of Firefox version numbers impacted by this problem.
* Create a workaround which allows the relevant HTML 5 date controls to be cleared another way (such as via JavaScript).
* Un-ignore the tests which are currently ignored

**Comment by craigfowler** [comment]
Migrated from https://github.com/csf-dev/CSF.Screenplay/issues/109

**Comment by craigfowler** [comment]
Kicked forward to milestone 0.3, because it will likely need a nontrivial JavaScript workaround. This should be blocked on #36 - when a mechanism for creating these will be produced.

**Comment by craigfowler** [comment]
Reopened - this is still a major problem.
